index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Treehouse Posters!</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+
+<!-- Poster 1 -->
+  <div class="row_top">
+    <article class="product_card">
+      <header class="product_name">
+        <h1>Guil #1</h1>
+      </header>
+          <section class="section_1">
```

Comment:
It's best to avoid naming classes based on order or position. If your layout changes later, you end up having to rename you classes, which has a ripple effect on your CSS.